source/_integrations/smartthings.markdown

```diff
@@ -285,88 +196,25 @@ The SmartThings Scene platform lets you activate scenes defined in SmartThings w
-
-#### Error: The token does not have the required OAuth scopes
-
-The personal access token entered is valid but does not have the required scopes as outlined in the prerequisites. Create a new token per the instructions in the prerequisites and try again.
-
-#### Error: SmartThings could not validate the webhook URL
-
-SmartThings was unable to reach your Home Assistant instance using the webhook URL. Enable debug logging to see the specific issue and follow the webhook troubleshooting checklist below.
-
-#### Aborted: There are no available SmartThings Locations
-
-This error message occurs when all of the SmartThings locations under the account linked to the personal access token are already setup in Home Assistant. Ensure you are using the correct personal access token or create an additional location in SmartThings to integrate and try again.
-
-#### Webhook Troubleshooting Checklist
+## Troubleshooting
 
-1. Ensure external URL is properly set to the _external address_ that Home Assistant is available to the internet. SmartThings must be able to reach this address.
-2. Validate there are no problems with your certificate or SSL configuration by using an online checker, such as [https://www.digicert.com/help/](https://www.digicert.com/help/).
-3. Some reverse proxy configuration settings can interfere with communication from SmartThings.  For example, TLSv1.3 is not supported.  Setting the supported cipher suite too restrictly will prevent handshaking. The following NGINX SSL configuration is known to work:
-   ```nginx
-   # cert.crt also contains intermediate certificates
-   ssl_certificate /path/to/cert.crt;
-   ssl_certificate_key /path/to/cert.key;
-   ssl_dhparam /path/to/dhparam.pem;
-   ssl_protocols TLSv1.2;
-   ssl_ciphers 'EECDH+AESGCM:EDH+AESGCM:AES256+EECDH:AES256+EDH';
-   ssl_prefer_server_ciphers on;
-   ssl_ecdh_curve secp384r1;
-   ssl_session_timeout  10m;
-   ssl_session_cache shared:SSL:10m;
-   ssl_session_tickets off;
-   ```
-4. While the error message (above) is being displayed, run the following command from outside your local network to confirm it is responding to the ping lifecycle event:
-    ```bash
-    curl -X POST https://{EXTERNAL_URL}/api/webhook/{WEBHOOK_ID} -H "Content-Type: application/json; charset=utf-8" -d $'{"lifecycle": "PING", "executionId": "00000000-0000-0000-0000-000000000000", "locale": "en", "version": "1.0.0", "pingData": { "challenge": "00000000-0000-0000-0000-000000000000"}}'
-    ```
-    Where `{EXTERNAL_URL}` is your external address and `{WEBHOOK_ID}` is the value of `webhook_id` from `.storage/smartthings` in your Home Assistant configuration directory.
+### Setting up the integration
 
-    The expected response is:
-    ```bash
-    {"pingData": {"challenge": "00000000-0000-0000-0000-000000000000"}}
-    ```
+#### What Client ID and Secret do I have to fill in?
```

Comment:
should this be part of the configuration section? The question probably comes up there.